src/components/AddPlaidBankAccount.js

```diff
@@ -72,6 +72,12 @@ const propTypes = {
 
     /** Additional text to display */
     text: PropTypes.string,
+
+    /** The OAuth URI + stateID needed to re-initialize the PlaidLink after the user logs into their bank */
+    receivedRedirectURI: PropTypes.string,
+
+    /** During the OAuth flow we need to use the plaidLink token that we initially connected with */
```

Comment:
This comment implies there is another token we should not be using. Is there? Would it be better to call this something like `plaidLinkOAuthToken`?

Reply:
yeah, there is not another token. I wanted to imply that when we reinitialize the `PlaidLink` we would not be creating a new `plaidLinkToken` but rather using the same one we linked with initially. 

I like the change to `plaidLinkOAuthToken`.